### DISCUSSION TITLE
User Accounts

### DISCUSSION THREAD

**Discussion by btihen** [hanamimastery/episodes#8] [Suggestions]
Perhaps there are preplanned tools like Devise in Rails - covering that would be helpful - as well well as some guidelines for a simple  (yet secure) do-it-yourself

**Comment by swilgosz** [comment]
Thanks! This is definitely on my plate, just waiting for the next alpha, where view helpers should be fully integrated and it will be easier to work with forms :).

For now I only coveref JWT authorization (without authentication) but will follow-up exploring the topic more.
